Source/DafnyCore/AST/Grammar/ProgramParser.cs

```diff
@@ -34,7 +34,7 @@ public class ProgramParser {
     CancellationToken cancellationToken) {
     var options = errorReporter.Options;
     var builtIns = new SystemModuleManager(options);
-    var defaultModule = new DefaultModuleDefinition(files.Where(f => !f.IsPreverified).Select(f => f.Uri).ToList());
+    var defaultModule = new DefaultModuleDefinition(files.Select(f => f.Uri).ToList());
```

Comment:
Bug fix: otherwise the contents of a .doo file passed directly without `--library` doesn't get compiled.